cactusbot/commands/magic/friend.py

```diff
@@ -0,0 +1,35 @@
+"""Manage friends."""
+
+from . import Command
+
+
+# TODO: Finish this command.
+class Friend(Command):
```

Comment:
Does not currently do anything. However, this requires the API as well, and can wait.